packages/mwp-app-server/src/util/getPlugins.js

```diff
@@ -1,15 +1,15 @@
 import Inert from 'inert';
-import CsrfPlugin from 'electrode-csrf-jwt';
+import CsrfPlugin from 'electrode-csrf-jwt/lib/csrf-hapi17';
 
 import config from 'mwp-config';
-import loggerPlugin from 'mwp-logger-plugin';
-import appRoutePlugin from 'mwp-app-route-plugin';
-import requestAuthPlugin from 'mwp-auth-plugin';
-import activityPlugin from 'mwp-tracking-plugin/lib/activity';
-import clickPlugin from 'mwp-tracking-plugin/lib/click';
-import languagePlugin from 'mwp-language-plugin';
-import serviceWorkerPlugin from 'mwp-sw-plugin';
-import apiProxyPlugin from 'mwp-api-proxy-plugin';
+import { plugin as loggerPlugin } from 'mwp-logger-plugin';
+import { plugin as appRoutePlugin } from 'mwp-app-route-plugin';
+import { plugin as activityPlugin } from 'mwp-tracking-plugin/lib/activity';
+import { plugin as clickPlugin } from 'mwp-tracking-plugin/lib/click';
+import { plugin as languagePlugin } from 'mwp-language-plugin';
+import { plugin as serviceWorkerPlugin } from 'mwp-sw-plugin';
+import { plugin as apiProxyPlugin } from 'mwp-api-proxy-plugin';
+import { plugin as requestAuthPlugin } from 'mwp-auth-plugin';
```

Comment:
any reason the plugin isn't the `default` export from all of these? seems like it would be a bit more intuitive and avoid the `plugin as xxx` renaming.

Reply:
```When written as a module, a plugin can either be top-level module export i.e module.exports = { register, name, version } or if you want your module to export more than a hapi plugin, it can be exported as exports.plugin = { register, name, version }.```

some of the plugins we export the register function or have a secondary export beyond just the plugin so for consistency / ease I wrote them all up the same. But I can change if you think it's better!

Reply:
ah interesting - I'm happy to stick to convention, although effectively that convention is there for commonJS compatibility. Still, probably best to stick to it :)